### ISSUE TITLE
Comparison to my implementation; small questions

### ISSUE THREAD

**Issue by czero69** [titu1994/keras-global-context-networks#3]
Hi, I find your github repo when trying to compare my tf GC implementation. I have a question about the sptial pooling in your code (it is what paper calls attention pool I guess?) I mean this line in your code:
```
context = torch.matmul(input_x, context_mask)
            # [N, C, 1, 1]
```
shouldn't matmul be followed by the division: for each c in context, c div by _broadcasted reduced sum of all context mask elements_? What do you think?
Like here, defined as "attention pooling": 
![Zrzut ekranu z 2020-08-21 11-42-47](https://user-images.githubusercontent.com/5120235/90887014-d7f48980-e3b3-11ea-9577-9c22abca9009.png)
Table1 from [paper](https://maigoakisame.github.io/papers/icassp19a.pdf)

**EDIT:** 

ok I think your implementation is ok :) In original [GC-paper](https://arxiv.org/pdf/1904.11492.pdf) it seems there is also no such division in equations. But I forgot there is Softmax before. I think div is not needed as Softmax will serve a similar job, as all weights after softmax will sum to 1 anyway.
